doc/src/devdocs/subarrays.md

```diff
@@ -152,7 +152,7 @@ we can define dispatch directly on `SubArray{T,N,A,I,true}` without any intermed
 Since this computation doesn't depend on runtime values, it can miss some cases in which the stride
 happens to be uniform:
 
-```julia
+```julia-repl
```

Comment:
Ideally we'd doctest a lot more in this file, but we'd need a way to run ordinary code within doctests. A named `@example` block sharing scope with doctests could be a way maybe, but Documenter doesn't support that at the moment afaict.